search/api_test.py

```diff
@@ -1,365 +1,95 @@
-"""
```

Comment:
probably best to view this on its own. its contents were moved to search/indexing_api_test.py. the contents here are new and unrelated